ndc-reference/bin/reference/main.rs

```diff
@@ -2353,7 +2353,7 @@ mod tests {
 
             let mut expected = mint.new_goldenfile(expected_path).unwrap();
 
-            let response_json = serde_json::to_string_pretty(&response.0).unwrap();
+            let response_json = serde_json::to_string_pretty(&response.0).unwrap() + "\n";
```

Comment:
Is this deliberate?

Reply:
We shouldn't be editing these by hand really, though. You can run `UPDATE_GOLDENFILES=1 cargo test` to update golden files when there is a breaking change.

Reply:
I have had trouble getting that command to work. (I highly recommend we switch to [Insta](https://insta.rs/).)